src/DataTable/Pagination.jsx

```diff
@@ -1,27 +1,27 @@
-import React from 'react'
+import React from "react";
 
-import Page from './Page'
+import Page from "./Page";
 
 const Pagination = ({ currentPageNumber, totalNumberOfPages, onChange }) => {
-  const pages =
```

Comment:
having too many lines hacky is anti clean code. so we should split the lines and give it names and verbs.